src/components/CategoryFilter/CategoryFilter.jsx

```diff
@@ -1,4 +1,10 @@
+import { useFilter } from "../../context/Filtercontext";
+
 const CategoryFilter = () => {
+  const { state, dispatch } = useFilter();
```

Comment:
you can give a better name for state and dispatch, something like filterState and filterDispatch